docs/vagrant.md

```diff
@@ -0,0 +1,134 @@
+   | `vagrant provision`                           | forces reprovisioning of the vagrant machine                                                                                         |
+   | `vagrant ssh`                                 | connects to machine via SSH                                                                                                          |
+   | `vagrant destroy`                             | stops and deletes all traces of the vagrant machine                                                                                  |
+   | `vagrant suspend`                             | Suspends a virtual machine (remembers state). Starts faster.                                                                         |
+   | `vagrant resume`                              | Resume a suspended machine (vagrant up works just fine for this as well)                                                             |
+   | `vagrant reload` --provision                  | Restart the virtual machine and force provis­ioning                                                                                  |
+   | `vagrant provision` --debug                   | Use the debug flag to increase the verbosity of the output                                                                           |
+   | `vagrant box list`                            | List the installed boxes                                                                                                             |
+   | `vagrant box remove` [virtualbox name]        | delete a box                                                                                                                         |
+   | `vagrant global­-status`                      | outputs status of all vagrant machines                                                                                               |
+   | `vagrant global­-status` --prune              | same as above, but prunes invalid entries                                                                                            |
+   | `vagrant up` --provision \| tee provision.log | Runs vagrant up, forces provisioning and logs all output to a file                                                                   |
+   | `VAGRANT_LOG=info`                            | Use the environement variable VAGRANT_LOG to set verbosity. Enter this in terminal session before you run the `vagrant up` command   |
+
+1. For help on vagrant commands, at a terminal session enter:
+
+   ```bash
+   vagrant -help
+   ```
+
+1. Once your VM environment is created, as an alternative to the vagrant commands listed above, you can use the `Oracle VM VirtualBox` application to manage the starting, stopping, and configuration of your VM. The `Help` menu provides detailed instructions on its use.
```

Comment:
Will using Virtualbox directly break the vagrant setup?

It seems unnecessary to mention this, when we can just use vagrant to manage our VMs in a consistent manner, regardless of provider.

Reply:
Opening virtualbox shouldn't break vagrant, sometimes that is useful for debugging what is going on. But I agree, this shouldn't be necessary in our docs

Reply:
I remove point 11.